bedrock/base/templates/includes/banners/pocket-pencil-banner.html

```diff
@@ -12,9 +12,9 @@
   {% set cta_text = 'See the winners' %}
 {% endif %}
 
-<aside class="pocket-banner c-banner hide-from-legacy-ie" id="pocket-banner">
+<aside class="monitor-plus-banner c-banner hide-from-legacy-ie" id="monitor-banner">
```

Comment:
I don't think these name changes should have been committed? Seems like the Monitor banner started life as a copy of the Pocket banner, but the original Pocket banner should be unchanged. These name changes break the Pocket banner styling.